packages/assets-controllers/src/NftController.ts

```diff
@@ -1297,25 +1276,6 @@ export class NftController extends BaseController<
     }
   }
 
-  // temporary method to get the correct chainId until we remove chainId from the config & the chainId arg from the detection logic
-  // Just a helper method to prefer the networkClient chainId first then the chainId argument and then finally the config chainId
-  #getCorrectChainId({
-    networkClientId,
-  }: {
-    networkClientId?: NetworkClientId;
-  }) {
-    if (networkClientId) {
-      const {
-        configuration: { chainId },
-      } = this.messagingSystem.call(
-        'NetworkController:getNetworkClientById',
-        networkClientId,
-      );
-      return chainId;
-    }
-    return this.#chainId;
-  }
```

Comment:
This has been removed and we are making a `call` to `NetworkController:getNetworkClientById` when needed since we no longer fallback to `this.#chainId`